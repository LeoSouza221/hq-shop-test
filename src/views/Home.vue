<template lang="pug">
  h1 teste
</template>

<script>
// @ is an alias to /src

export default {
  name: 'Home',
};
</script>
