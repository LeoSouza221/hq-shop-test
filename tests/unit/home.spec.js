import { mount } from '@vue/test-utils';
import Home from '@/views/Home.vue';

test('home match with the snapshot', () => {
  const wrapper = mount(Home);

  expect(wrapper.element).toMatchSnapshot();
});

test('have a teste text', () => {
  const wrapper = mount(Home);

  expect(wrapper.text()).toBe('teste');
});

