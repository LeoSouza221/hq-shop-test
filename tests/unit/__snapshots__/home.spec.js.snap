// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home match with the snapshot 1`] = `
<h1>
  teste
</h1>
`;
